script.js

```diff
@@ -1,6 +1,77 @@
-function runProgram(){
+function getQuestions() {
+    const questions = document.querySelectorAll(".question");
+    const chapter = `chapter=${questions[0].value}`;
+    const abilities = `abilities=${questions[1].value}`;
+    getStands(chapter, abilities);
+}
+
+async function getStands(chapter, abilities) {
+    const infoBoxes = document.querySelectorAll(".stat");
+    const standImage = document.querySelector("#standImage");
+    for (let infoBox of infoBoxes) {
+        infoBox.textContent = "";
+    }
+    standImage.src = "";
+    
+    let standData = null;
+    const standURL = "https://stand-by-me.herokuapp.com/api/v1/stands/query/query?";
+    try { 
+        const response = await fetch(standURL + chapter + "&" + abilities);
+        standData = await response.json();
+    }
+    catch(error) {
+        console.log(error);
+    }
+    if (standData.length === 0) {
+        const nameBox = document.querySelector("#name");
+        nameBox.textContent = "You didn't get a stand! You died!";
+    }
+    else {
+        getOneStand(standData);
+    }
+}
+function getOneStand(data) {
+    let randomChance = Math.floor(data.length * Math.random());
+    let yourStand = data[randomChance];
+    showStats(yourStand);
+    showUser(yourStand);
+}
```

Comment:
Overall, these are some really good, succinct and well-decomposed functions! Nice job